x-pack/metricbeat/tests/system/xpack_metricbeat.py

```diff
@@ -1,10 +1,9 @@
+import metricbeat
 import os
 import sys
 
 sys.path.append(os.path.join(os.path.dirname(__file__), '../../../../metricbeat/tests/system'))
 
-import metricbeat
```

Comment:
And here, please don't change this file, this import requires to modify the python path first.

Reply:
All this changes happened because after running `mage fmt update`, but I'll revert all of them